website/docs/containers/registries/index.md

```diff
@@ -2,18 +2,275 @@
 sidebar_position: 3
 title: Registries
 description: Working with container registries
-tags: [podman-desktop, containers, registries]
+tags: [podman-desktop, containers]
 keywords: [podman desktop, podman, containers, registries]
 hide_table_of_contents: false
+image: /img/docs/containers/registries/img/registries.png
 ---
 
-# Working with container registries
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-Podman Desktop has access to public registries by default.
+# Setting up container registries
 
-You might want to configure Podman Desktop to:
+## Overview
```

Comment:
to simplify and make everybody happy, let's remove this "intro" section and get directly into the content.
cc @deboer-tim , @benoitf , @themr0c

Reply:
`Overview`for sure and mostly `Before you start`

Reply:
Removed.